bemani/backend/iidx/cannonballers.py

```diff
@@ -1,15 +1,2184 @@
+                    10024,
+                ],
+            },
+            {
+                'name': 'FEAT. KANAE ASABA',
+                'id': 3,
+                'open_music_id': 24101,
+                'songs': [
+                    25056,
+                    23033,
+                    24011,
+                    21034,
+                ],
+            },
+            {
+                'name': 'BLOOM',
+                'id': 4,
+                'songs': [
+                    15054,
+                    20053,
+                    20004,
+                    23083,
+                ],
+            },
+            {
+                'name': 'VACUUM',
+                'id': 5,
+                'songs': [
+                    19070,
+                    23021,
+                    17002,
+                    18071,
+                ],
+            },
+            {
+                'name': 'PRINCESS',
+                'id': 6,
+                'open_music_id': 12100,
+                'songs': [
+                    25024,
+                    20021,
+                    19046,
+                    12002,
+                ],
+            },
+            {
+                'name': 'TECHNO',
+                'id': 7,
+                'open_music_id': 19100,
+                'songs': [
+                    25026,
+                    19063,
+                    17061,
+                    23085,
+                ],
+            },
+            {
+                'name': 'POP',
+                'id': 8,
+                'open_music_id': 15103,
+                'songs': [
+                    15061,
+                    20099,
+                    20101,
+                    22093,
+                ],
+            },
+            {
+                'name': 'CARDINAL',
+                'id': 9,
+                'open_music_id': 13101,
+                'songs': [
+                    13038,
+                    13026,
+                    13000,
+                    13010,
+                ],
+            },
+            {
+                'name': 'O2K',
+                'id': 10,
+                'songs': [
+                    21059,
+                    19002,
+                    22054,
+                    16020,
+                ],
+            },
+        ]
+
+        # Secret course definitions
+        secret_courses: List[Dict[str, Any]] = [
+            {
+                'name': 'EMOTIONAL',
+                'id': 1,
+                'songs': [
+                    21009,
+                    25006,
+                    17065,
+                    25066,
+                ],
+            },
+            {
+                'name': 'RYU',
+                'id': 2,
+                'songs': [
+                    23082,
+                    21007,
+                    23037,
+                    25067,
+                ],
+            },
+        ]
+
+        # For some reason, omnimix crashes on course mode, so don't enable it
```

Comment:
Is that still the case with CB omni? They seriously never fixed it?

Reply:
i went and downloaded cb omnimix just to check this